examples/tidyverse/Dockerfile

```diff
@@ -0,0 +1,8 @@
+
+FROM rhub/r-minimal
+
+RUN installr -d \
+   -t "R-dev file automake autoconf linux-headers" \
+   -a "libxml2-dev icu-libs" `# needed for xml2 and stringr`  \
```

Comment:
I don't think we need libxml2-dev on the image, just the lib itself, so you can add `libxml2-dev` to the `-t` line and have `libxml` in `-a`.

Reply:
Ah, thanks, will do!